### PR TITLE
DM-36096: Add some text on the Campaign Management Board

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

**Review by womullan — APPROVED** [review]
thanks that is good

File: management.tex [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+The status of the epics and how they related to the relevant milestones was monitored as part of the weekly coordination or DPLT meetings.
+
+\subsection{Change Control}
+
+Data Preview 0.2 used v23.0.x of the LSST Science Pipelines Software and that was derived from a weekly release from September 2021 (\texttt{w.2022.40}).
+We decided to group processing into distinct ``steps'' that allowed updates to the software used in later stages of processing to be worked on whilst earlier steps were executing.
+
+We continued to want to use the v23 release for all data processing and that required that we had a process to determine which patches would need to be back-ported to the release branch as needed before each step could begin.
+
+The Data Management Change Control Board (DMCCB) and DPLT delegated authority to a new Campaign Management Board that had the following membership:
```
Comment (timj) [inline management.tex:50]: Looking at confluence we seemingly called it the Data Release Steering Committee (I was going off our slack channel).

File: management.tex [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+
+\subsection{Oversight}
+
+The Data Production Leadership Team (DPLT) consisted of representatives from all the teams involved in the data preview process as well as the data facilities.
+The DPLT met fortnightly to discuss any issues and minutes were recorded on Confluence.
+
+The membership of the DPLT was:
+
+\begin{itemize}
+\item Wil O'Mullane
```
Comment (timj) [inline management.tex:13]: Somehow missing my name off here...

File: management.tex [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+
+\item A request is made that a ticket should be applied to the release branch by applying a \texttt{backport-v23} tag to the Jira ticket.
+\item The board would then discuss the relative merits of the back-port and if approved a \texttt{backport-approved} label would be added.
+\item The work on the back-port would then be scheduled by the relevant T/CAM following instructions in the developer guide.\footnote{\url{https://developer.lsst.io/work/backports.html}}
+\item Once the code is on the \texttt{v23.0.x} branch a \texttt{backport-done} label would be applied.
+
+\end{enumerate}
+
+A Jira query was constructed to find all the tickets and track their porting status.
+There were 64 tickets with back-port requests and 61 of those were approved and applied to the release branch.
```
Comment (timj) [inline management.tex:74]: Turns out that when a ticket was rejected we removed the label so it's now not easy to find out how many we rejected. @ctslater pointed this out to me and I had forgotten. The three described here were ones that we couldn't decide whether to do or not...

**Pull request merged**